node/cmd/sentinel/main.go

```diff
@@ -6,17 +6,17 @@ import (
 	"os"
 	"sync"
 
-	"bisonai.com/orakl/node/pkg/checker/balance"
-	// "bisonai.com/orakl/node/pkg/checker/dal"
-	// "bisonai.com/orakl/node/pkg/checker/dalstats"
-	"bisonai.com/orakl/node/pkg/checker/dbcronjob"
-	"bisonai.com/orakl/node/pkg/checker/event"
-	"bisonai.com/orakl/node/pkg/checker/health"
-	"bisonai.com/orakl/node/pkg/checker/inspect"
-	"bisonai.com/orakl/node/pkg/checker/offset"
-	"bisonai.com/orakl/node/pkg/checker/peers"
-	"bisonai.com/orakl/node/pkg/checker/signer"
-	"bisonai.com/orakl/node/pkg/logscribeconsumer"
+	"bisonai.com/miko/node/pkg/checker/balance"
+	// "bisonai.com/miko/node/pkg/checker/dal"
+	// "bisonai.com/miko/node/pkg/checker/dalstats"
```

Comment:
it was temporarily commented since it was suspected to be the cause of pgsql high cpu usage, turns out to be not the case, will be rollbacked(https://github.com/Bisonai/orakl/pull/2171)